apps/docs/content/guides/functions/debugging-tools.mdx

```diff
@@ -0,0 +1,23 @@
+---
+id: 'functions-debugging-tools'
+title: 'Debugging Tools'
+description: 'Debugging Tools for Edge Functions.'
+subtitle: 'Debugging Tools for Edge Functions.'
+---
+
+Since [v1.171.0](https://github.com/supabase/cli/releases/tag/v1.171.0) the Supabase CLI supports debugging Edge Functions via the v8 inspector protocol, allowing for debugging via [Chrome DevTools](https://developer.chrome.com/docs/devtools/) and other Chromium-based browsers.
+
+### Inspect with Chrome Developer Tools
+
+You can use the [Chrome DevTools](https://developer.chrome.com/docs/devtools/) to set breakpoints and inspect the execution of your Edge Functions.
+
+1. Serve your functions in [inspect mode](/docs/reference/cli/supabase-functions-serve): `supabase functions serve --inspect-mode brk`. This will set a breakpoint at the first line to pause script execution before any code runs.
```

Comment:
I merged the PR to make this the default. Should we update here to say `--inspect`? https://github.com/supabase/cli/pull/2370/files

Reply:
But it'll be a while until it's released, right?